web/src/main/kotlin/com/example/ajaxproject/nats/GetUserAllNatsController.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.ajaxproject.nats
+
+import com.example.ajaxproject.NatsSubject.GET_ALL_USERS
+import com.example.ajaxproject.UserOuterClass.GetAllUsersRequest
+import com.example.ajaxproject.UserOuterClass.GetAllUsersResponse
+import com.example.ajaxproject.UserOuterClass.UserList
+import com.example.ajaxproject.repository.UserRepository
+import com.google.protobuf.Parser
+import io.nats.client.Connection
+import org.springframework.data.domain.PageRequest
+import org.springframework.stereotype.Component
+import reactor.core.publisher.Mono
+
+@Component
+class GetUserAllNatsController(
+    private val userMapper: UserMapper,
+    private val userRepository: UserRepository,
+    override val connection: Connection,
+) : NatsController<GetAllUsersRequest, GetAllUsersResponse> {
+
+    override val subject: String = GET_ALL_USERS
+
+    private companion object {
```

Comment:
Must be in the end of class

Reply:
![image](https://github.com/OleksandRomaniuk/Ajax/assets/116563306/2087be46-5e33-4536-b816-736f44ea0285)

https://kotlinlang.org/docs/coding-conventions.html#class-layout

Reply:
Moved it to bottom